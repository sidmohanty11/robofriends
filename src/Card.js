import React from 'react';
import 'tachyons';

function Card({username, name, email, id}) {
	return(
		<div className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5">
			<img alt="" src={`https://robohash.org/${id}?200x200`} />
			<div>
				<h2>{username}</h2>
				<p>name: {name}</p><p>email: {email}</p>
			</div>
		</div>
	);
}

export default Card;