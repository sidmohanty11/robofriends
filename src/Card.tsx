import React from 'react';

type robot = {
	id: number,
	username: string,
	email: string,
	name: string,
}

function Card({username, name, email, id}: robot) {
	return(
		<div className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5">
			<img alt="" src={`https://robohash.org/${id}?size=200x200`} />
			<div>
				<h2>{username}</h2>
				<p>name: {name}</p><p>email: {email}</p>
			</div>
		</div>
	);
}

export default Card;