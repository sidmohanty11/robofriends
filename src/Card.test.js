import { shallow } from 'enzyme';
import Card from './Card';

it("expect if component is present", () => {
    expect(shallow(<Card />)).toMatchSnapshot();
});