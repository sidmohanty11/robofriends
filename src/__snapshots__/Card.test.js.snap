// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect if component is present 1`] = `
<div
  className="bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt=""
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p>
      name: 
    </p>
    <p>
      email: 
    </p>
  </div>
</div>
`;
